components/ProgressIndicator/__tests__/ProgressIndicator.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import createSnapshotTest from 'test-utils/createSnapshotTest';
+
+import ProgressIndicator from '../ProgressIndicator';
+
+describe('ProgressIndicator', () => {
```

Comment:
looks like you've gotta edit the tests!

Reply:
do you want me to use jest snapshot testing, I saw another issue about moving to react-testing-library?

Reply:
I think snapshot tests should work fine here! We have a utility for that though, look at other test files (like `Button.test.js`)